.editorconfig

```diff
@@ -0,0 +1,19 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+trim_trailing-whitespace = true
+insert_final_newline = true
+
+[*.{glsl,rs,toml}]
+indent_style = space
+indent_size = 4
+
+[*.yml]
+indent_style = space
+indent_size = 2
+
+[Makefile]
+indent_style = tab
+indent_size = 8
```

Comment:
What's your argument for using such a high indent size for makefiles? I'd stick to 4.